tests/cli/test_state.py

```
@@ -0,0 +1,154 @@
+        from typer.testing import CliRunner
+
+        from cli.state import CLIState, _get_state
+
+        results: list[bool] = []
+
+        test_app = typer.Typer()
+
+        @test_app.callback()
+        def cb(
+            ctx: typer.Context,
+            verbose: bool = typer.Option(False, "--verbose"),
```

Ruff (0.15.10)

[warning] Boolean-typed positional argument in function definition

(FBT001)